collections.md

```diff
@@ -155,6 +155,151 @@ The json representation is as follow
 ```
 see also the [ResourcePolicies endpoint](resourcepolicies.md)
 
+### Collection Harvesting Settings
+**GET /api/core/collections/<:uuid>/harvestingSettings**
+
+It returns the harvesting settings for the current collection. This information is only accessible for users with collection administration permissions
+
+The harvest_type can be any of:
+* NONE
+* METADATA_ONLY
+* METADATA_AND_REF
+* METADATA_AND_BITSTREAMS
+
+The harvest_status can be any of:
+* READY
+* BUSY
+* QUEUED
+* OAI_ERROR
+* UNKNOWN_ERROR
+
+The metadata_config_id can be one of the ids from [Harvesting metadata configuration Endpoints](harvesting_metadata_configs.md)
+
+A sample json response:
+
+```json
+{
+  "harvest_type": "METADATA_ONLY",
+  "oai_source": "https://dspace.mit.edu/oai/request",
+  "oai_set_id": "col_1721.1_114174",
+  "harvest_message": null,
+  "metadata_config_id": "dc",
+  "harvest_status": "READY",
+  "harvest_start_time": null,
+  "last_harvested": null,
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/core/collections/6f944500-c300-449a-9023-a5ad8bd21160/harvestingSettings"
+    }
+  },
+  "_embedded": {
+    "metadata_configs": {
```

Comment:
Is there a reason for embedding all `metadata_configs` on a simple `GET`?  It seems like those are also available on the other endpoint, so I'm wondering whether they need embedding here?

Reply:
Embedding is never required, but when the harvesting settings are requested, the list of potential metadata configs is typically relevant as well
Since it's only info from a config file, it's a very low-load detail to embed compared to the added value of reducing the amount of requests

Reply:
@benbosman : Ok, that's fine.  One note here is that we likely should correct the name here from `metadata_configs` to `harvestermetadata` (as that's the new `self` link path)

Reply:
I've updated this now